app/controllers/venues_controller.rb

```diff
@@ -0,0 +1,5 @@
+class VenuesController < ApplicationController
```

Comment:
try to keep your commit messages pertinent to the value you're delivering.

A good commit message here would be:

`Implement venue listing`